front/src/applications/operationalStudies/hooks/useAutoUpdateProjection.ts

```diff
@@ -36,7 +35,6 @@ const useAutoUpdateProjection = (
   const dispatch = useAppDispatch();
   const currentTrainIdForProjection = useSelector(getTrainIdUsedForProjection);
   const selectedTrainId = useSelector(getSelectedTrainId);
-
```

Comment:
Why remove this empty line ? It makes the code more difficult to read

Reply:
It's surely a mistake, i probably put something here at some time and removed it, i'll will add an empty line